### PR TITLE
I165 export word doc

### PR DESCRIPTION
**Pull request by sarahrainbow**
**Pull Request request related to  #165**      

**Describe what the PR does**    
* implements exporting programme script to word document

**State whether the PR is ready for review or whether it needs extra work**    
   
Currently the export includes clip name and timecode references for text selections. This approach can be tested in upcoming research to see to see if any further modifications need to be made.

## Acceptance Criteria

- [ ] Ability to generate a Microsoft Word version of Programme Script
- [ ] Word document includes:
  - [ ] paper cuts
  - [ ] headings
  - [ ] notes
  - [ ] voice notes

### CODE REVIEW

File: src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/json.js [hunk 1]
```diff
@@ -29,15 +29,18 @@ const formatToJSONEvent = (transcript, element) => {
...
-  // alert('this function has not been implemented yet');
+const getJson = (title, elements, transcripts, isWordDoc) => {
   return elements.reduce((res, element) => {
     if (element.type === 'paper-cut') {
       const transcript = getCurrentTranscript(element, transcripts);
       const jsonEvent = formatToJSONEvent(transcript, element);
       res.index += 1;
       jsonEvent.id = res.index;
       res.events.push(jsonEvent);
+    } else if (isWordDoc) {
```
Comment (allishultes) [inline src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/json.js:40]: Nice work adding the non-transcript elements :)

File: src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/programme-script-json-to-docx.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import { Document, HeadingLevel, AlignmentType, Paragraph, TextRun, Packer } from 'docx';
+import { shortTimecode } from 'slate-transcript-editor';
+import downloadjs from 'downloadjs';
+
+const programmeScriptJsonToDocx = (edlJson, title, isWithClipReference) => {
```
Comment (allishultes) [inline src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/programme-script-json-to-docx.js:5]: Really small thing on naming - could 'isWithClipReference' be something like 'includeClipReference' or something slightly more clear?

**Review by allishultes — APPROVED** [review]
LGTM and runs fine locally :) Nice work!

File: src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/programme-script-json-to-docx.js (hunk 2 above)
Comment (sarahrainbow) [reply src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/programme-script-json-to-docx.js]: good shout - all done 👍

**Comment by aniablaziak** [comment]
looks good and works perfect 🌟 

this is probably a case for another ticket but initially I tried to export transcript from this screen and noticed MS Word export isn't working (the others seem to work ok). do we need to implement this? probably a question for @emettely 

Screenshot 2021-04-12 at 15 58 40

**Comment by emettely** [comment]
> looks good and works perfect 🌟
> 
> this is probably a case for another ticket but initially I tried to export transcript from this screen and noticed MS Word export isn't working (the others seem to work ok). do we need to implement this? probably a question for @emettely
> 

Ah interesting, probably do not need to implement it :)

**Pull request merged**